crates/ruma-client-api/src/account.rs

```diff
@@ -44,9 +44,7 @@ impl<'a> IdentityServerInfo<'a> {
 }
 
 /// Possible values for deleting or unbinding 3PIDs.
-///
-/// This type can hold an arbitrary string. To check for formats that are not available as a
-/// documented variant here, use its string representation, obtained through `.as_str()`.
+#[doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/doc/string_enum.md"))]
```

Comment:
To make sure there's a new paragraph:

```suggestion
///
#[doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/doc/string_enum.md"))]
```

Reply:
It's removed by rustfmt

Reply:
Oh, I remember coming across that.